package.json

```
@@ -46,7 +46,7 @@
     "typescript": "^4.2.3"
   },
   "engines": {
-    "node": "12"
+    "node": ">= 12"
```

I think we don't want to support 13?

```suggestion
    "node": "12 | >= 14"
```